### PR TITLE
Create ISSUE_TEMPLATE.md

### PR DESCRIPTION
**Pull request by JoviDeCroock**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+<!--
+🚨HOLD UP PLEASE 🚨
```
Comment (marvinhagemeister) [inline .github/ISSUE_TEMPLATE.md]: I'm a little worried that it scares of users who have never opened an issue on GitHub 🙈

Comment (JoviDeCroock) [reply .github/ISSUE_TEMPLATE.md]: It sais please :( I'm scared about people not reading it... What would you suggest?

Comment (ForsakenHarmony) [inline .github/ISSUE_TEMPLATE.md]: There's a space between the emoji and the text at the end but not at the start :eyes: (sorry)

**Comment by marvinhagemeister** [comment]
Love the new template 👍

**Pull request merged**